lighthouse-extension/app/src/popup.js

```diff
@@ -45,17 +44,38 @@ function getLighthouseCommitHash() {
 }
 
 function getChromeVersion() {
+  // @ts-ignore
   return /Chrome\/([0-9.]+)/.exec(navigator.userAgent)[1];
 }
 
 function showRunningSubpage() {
-  document.querySelector('.status').classList.add(subpageVisibleClass);
+  find('.status', document).classList.add(subpageVisibleClass);
 }
 
 function hideRunningSubpage() {
-  document.querySelector('.status').classList.remove(subpageVisibleClass);
+  find('.status', document).classList.remove(subpageVisibleClass);
 }
 
+/**
+ * Guaranteed context.querySelector. Always returns an element or throws if
+ * nothing matches query.
+ * @param {string} query
+ * @param {ParentNode} context
+ * @return {HTMLElement}
+ */
+function find(query, context) {
```

Comment:
we seem to always use `document`, do we need context?

Reply:
added default of `document`. WDYT?

Reply:
sure that sg 👍